examples/resources/awscc_resiliencehub_app/resiliencehub_app.tf

```diff
@@ -0,0 +1,34 @@
+resource "awscc_resiliencehub_app" "main" {
+  name             = "test-app"
+  resource_mappings = [ {
+      physical_resource_id = {
+        identifier = "s3://terraform-state-files/test-app.tfstate"
+        type = "Native"
+        # Optional aws_region = ""
+        # Optional aws_account_id = ""
+      }
+      mapping_type = "Terraform"
+      # Optional  resource_name = ""
+      #           resource_type = ""
+      #           logical_resource = { 
+      #             logical_resource_name = ""
+      #             resource_mapping_name = ""
+      #             resource_mapping_arn = awscc_resiliencehub_app.app.arn 
+      }]
+  resiliency_policy_arn = "arn:aws:resiliencehub:us-west-1:<account-id>:resiliency-policy/<id>"
+  app_template_body = <<EOF
```

Comment:
Would suggest replacing the heredoc with jsonencode.

Reply:
Done.